listenbrainz_spark/recommendations/recording/recommend.py

```diff
@@ -87,9 +87,21 @@ def process_recommendations(recommendation_df, limit):
               FROM recommendation
         )
         SELECT user_id
-             , recording_mbid
-             , score
-             , date_format(latest_listened_at, "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") AS latest_listened_at
+             , array_sort(
```

Comment:
This is change is not to do with raw_recs but it felt to me that collecting in a list in python was slower and doing it in spark seemed somewhat faster.